From 46d3be8026baf79c87e81eae1a7faa42ff76b944 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Thu, 19 Apr 2018 15:06:30 +0800
Subject: [PATCH] Revert "arm64: dts: rockchip: rk3308-evb-v10: Add
 regulator-early-min-microvolt for core"

This reverts commit 602e4a09691e24def020709bb59b6bb4a10847ea.

Change-Id: If70202a1f8b6837c8804f054e0350b5a63c2e6c2
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
index 9f83a2c7207e..c731d4c0ca69 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
@@ -121,7 +121,6 @@
 		regulator-min-microvolt = <827000>;
 		regulator-max-microvolt = <1340000>;
 		regulator-init-microvolt = <1015000>;
-		regulator-early-min-microvolt = <1015000>;
 		regulator-always-on;
 		regulator-boot-on;
 		regulator-settling-time-up-us = <250>;
-- 
2.35.3

